doc/_data/support/federation.yml

```diff
@@ -1,5 +1,12 @@
 # Data that is used to generate our federation page.
 
+- url_binderhub: https://2i2c.mybinder.org
+  logo: https://2i2c.org/media/logo.svg
```

Comment:
@choldgraf is [this](https://2i2c.org/media/logo.svg) the right logo URL to use?